API/models/schemas/refs/types.json

```diff
@@ -2,34 +2,48 @@
     "$id": "refs/types.json",
     "definitions": {
         "color": {
-            "pattern": "^[0-9a-fA-F]{6}$"
+            "pattern": "^[0-9a-fA-F]{6}$",
+            "description": "a 6 length hex value (e.g. 'C0FFEE')"
+        },
+        "colorTemplate": {
+            "pattern": "(^[0-9a-fA-F]{6}$)|(^@[a-zA-Z0-9-]+$)|(^$)",
+            "description": "a 6 length hex value or '@[color name]' (e.g. 'C0FFEE', '@blue')"
         },
         "metricImperialUnit": {
-            "enum": ["mm", "cm", "m", "f"]
+            "enum": ["mm", "cm", "m", "f"],
+            "description": "'mm', 'cm', 'm' or 'f'"
```

Comment:
This one is not replaced in the error message (not a pattern, no pattern error substring), but the error message for enums is already ok (example: /attributes/posXYUnit value must be one of "m", "t", "f"), so no need to replace it.

Reply:
I've removed the unnecessary descriptions, since the error messages are already clear.